versions/3.0.3.md

```diff
@@ -2299,7 +2299,7 @@ The following properties are taken directly from the JSON Schema definition and
 - exclusiveMinimum
 - maxLength
 - minLength
-- pattern (This string SHOULD be a valid regular expression, according to the [ECMA 262 regular expression](https://www.ecma-international.org/ecma-262/5.1/#sec-15.10.1) dialect)
+- pattern (This string SHOULD be a valid regular expression, according to the [Ecma-262 Edition 5.1 regular expression](https://www.ecma-international.org/ecma-262/5.1/#sec-15.10.1) dialect)
```

Comment:
Conflict with link fragment change resolved here. Now LGTM.

Reply:
Thanks Mike. I discussed with @webron , too, and ready to merge.